src/About.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import Navbar from './Components/Navbar/Navbar'
+import './about.css'
+
+function About() {
+  return (
+    <>
+    <Navbar/>
+    <div className='aboutpage'>
+    <img src='https://i1.sndcdn.com/avatars-000650172765-896cr8-t500x500.jpg' className='lvximg' />
```

Comment:
Avoid using url in your components, create variables and use in place of url here.